### DISCUSSION TITLE
Using a CloudFlare worker. Confused on how to craft my snippet.

### DISCUSSION THREAD

**Discussion by perfectly-preserved-pie** [plausible/analytics#2422] [Self-Hosted Support]
I'm following this guide:

https://plausible.io/docs/proxy/guides/cloudflare

I'm not getting any visitor hits on the dashboard.
My Plausible instance is hosted at plausible.domainA.com. The site I want to track analytics for is hosted at domainB.com. I own both domains.

What should my snippet look like? Right now I have

`<script defer data-domain="plausible.domainA.com" data-api="https://domainA.workers.dev/myproject/event" src="https://domainA.workers.dev/myproject/script.js" type="application/javascript"></script>`

I see the snippet in the HTML <head> tag when I use the browser tools, and I see it successfully get fetched in the Network tab (200 status code). But I still get the little green dot, no stats.

domainB.com is a Dash app running on a firewalled server via an Nginx reverse proxy, but ports 80 & 443 are both open and unrestricted. Any ideas??

**Comment by DejavuMoe** [comment]
I recommend using Cloudflare's rewrite rules for this rewrite process (if you use Cloudflare CDN), I wrote [an article](https://dejavu.moe/posts/plausible-selfhosted-with-docker-complete-guide/#重写路径建议) about this operation and it worked fine for me.

BTW, can be read using your browser's automatic translation if you speak other language 🤤
